interview-questions.md

```diff
@@ -7,62 +7,61 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
-
-  Researched answer:
+  Your answer: Conditional statements add logic to a function. This logic allows variables to be evaluated by the criteria set forth and return an output if the criteria was met or not met. If, else if, and else statements are examples of conditional statements. If x > 32, console.log("The temperature is above freezing."); else if x === 32, console.log("The temperature is freezing."); else console.log("The temperature is below freezing.")
 
+  Researched answer: Conditional statements perform different actions based on its Boolean value of true or false. From the Boolean value, different outcomes are possible. We can use comparison operators to test these conditions inside of our if/else statements.
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is the technology used for coding that includes a multitude of commands for the terminal, for pushing and pulling from local to remote, for creating new, files, and so much more. Github is the platform for which git technology is often used. We use Github to actually push and pull files, code, pair program.
 
-  Researched answer:
+  Researched answer: Git is a free, open-sourced version control system that allows for collaborative work on the same code. Github is a graphical user interface (GUI) that is a hosting service for git repositories. 
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: The && (and) operator and the || (or) operator. With the && operator, both statements have to be true in order to return the Boolean value of true. Forthe || operator, only one statement has to evaluate to true to return the value of true, and if the first statement is true, it may not run the second statement.
 
-  Researched answer:
+  Researched answer: The || operator can be used with operands that aren't Boolean, but the return will be converted to a Boolean primitive. The && operator has higher precedence than the || operator, which means it will execute first.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: An index is the location of an element in an array and has a 0-base. For example, myArray = ["apple", "banana", "strawberry"]. Here, index[0] = apple. index[1] = "banana", and index[2] = "strawberry".  A value is what a something is equal too such as a variable. For example, temp1 = 75, 75 would be the value of temp1 as there is no location and no index of temp1.
```

Comment:
I really like your example!  In that instance, the index is the number inside the square bracket and the value is the content in that location.  Ex: Index is 0 and the value at index 0 is "apple"